### PR TITLE
Stop otbr-agent for occupied dongle device process

### PR DESCRIPTION
**Pull request by bermanRon**
Stop otbr-agent for occupied dongle device process
https://github.com/Samsung/ioter/issues/94

### CODE REVIEW

File: script/run [hunk 1]
```diff
@@ -17,6 +17,11 @@ kill_allprocess()
     done
 }
 
+kill_otbr()
+{
+    sudo systemctl stop otbr-agent
```
Comment (junyong-sim) [inline script/run:22]: It there is no `otbr-agent` installed on linux pc, the above command will return a error.
How about redirecting standard error to `/dev/null` ?
For example,
`sudo systemctl stop otbr-agent 2>/dev/null`

Comment (spdkimo) [reply script/run]: Plus, reading the process status is required. No need to kill it always.

Comment (junyong-sim) [reply script/run]: I think reading the process status make the script complicated. So, I think it would be good to keep it as current status.

**Review by junyong-sim — APPROVED** [review]
LGTM

**Comment by sangsoolee** [comment]
@bermanRon 
Thanks for contribution.

**Pull request merged**